main/src/ca/uwaterloo/flix/language/ast/TokenKind.scala

```diff
@@ -307,6 +339,8 @@ object TokenKind {
 
   case object Tilde extends TokenKind
 
+  case object TildeTilde extends TokenKind
```

Comment:
What's this?

Reply:
Unary Type operator `~~` for case complement 
https://github.com/flix/flix/blob/bf14c3ac86d24e9df07b54e8c5257999424e5794/main/src/ca/uwaterloo/flix/language/phase/Parser.scala#L1575

Reply:
Right.

Reply:
We don't know our own language :)